Formula/vlang.rb

```diff
@@ -26,14 +26,15 @@ class Vlang < Formula
     # "[v:master] {short SHA of the vlang release commit} - {vlang version number}".
     # The sources of this V compiler commit need to be used here
     url "https://github.com/vlang/vc.git",
-        revision: "fd5f57740ff6d7a8566b774318df54c2fa460f92"
+        revision: "5dc8384638f515a098f75296fa7123c5e6a4964a"
   end
 
   def install
-    resource("vc").stage do
-      system ENV.cc, "-std=gnu11", "-w", "-o", buildpath/"v", "v.c", "-lm"
-    end
-    system "./v", "self"
+    vc = buildpath/"vc"
+    vc.install resource("vc")
+    (buildpath/"thirdparty/tcc/tcc.exe").write_exec_script ENV.cc
```

Comment:
Looks like this doesn't work. Should we just add the `tccbin` resource here? I feel like there should be a way not to have to...